src/app/shared/more-description/more-description.component.ts

```diff
@@ -16,52 +16,42 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 */
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, AfterViewInit, ElementRef, ViewChild } from '@angular/core';
 import { UserSettingsHandlerService } from '@mdm/services/utility/user-settings-handler.service';
-import { MarkdownParserService } from '@mdm/utility/markdown/markdown-parser/markdown-parser.service';
 
 @Component({
   selector: 'mdm-more-description',
   templateUrl: './more-description.component.html',
   styleUrls: ['./more-description.component.sass']
 })
-export class MoreDescriptionComponent implements OnInit {
+export class MoreDescriptionComponent implements OnInit, AfterViewInit {
   @Input() description: string;
   @Input() length: any;
 
-  maxLength = 100;
-  showMore = false;
-  shortDesc: string;
-  fullDesc: string;
+  showMore = true;
+  isOverflowing = false;
+  @ViewChild('descriptionContent', { static: false })
+  descriptionContent: ElementRef;
+
   constructor(
-    userSettingsHandler: UserSettingsHandlerService,
-    private markdownParser: MarkdownParserService
+    userSettingsHandler: UserSettingsHandlerService
   ) {
     this.showMore = userSettingsHandler.get('expandMoreDescription');
   }
 
   ngOnInit() {
```

Comment:
If `ngOnInit()` is empty now, it can just be removed along with the `OnInit` interface.

Reply:
I've now made this change